frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -9,45 +9,33 @@ import { useNavigate } from 'react-router-dom'
 import { MissionControlButtons } from './MissionControlButtons'
 import BatteryStatusView from '../RobotCards/BatteryStatusView'
 import { BatteryStatus } from 'models/Battery'
+import { MissionRobotDisplay } from './MissionRobotDisplay'
 
 interface MissionProps {
     mission: Mission
 }
 
 const StyledMissionCard = styled(Card)`
-    width: 432px;
-    padding: 10px;
+    width: 372px;
+    padding: 12px;
 `
 
```

Comment:
There is a newline between StyledMissionCard and StyledTitle, but not between the other styles. Couold you also fix that?